tests/specs/test-access-control.js

```diff
@@ -56,10 +57,10 @@ describe('Access Control Page - Verify that', function() {
 
         });
         accessControlPage.submit_token(valid_token);
-        loginpage = browser.element('h2=Installed snaps');
-        loginpage.waitForVisible();
-        expect(loginpage.getText(), "Login Failed with valid token").to.contain('Installed snaps');
-
+        var snaplist = homePage.snaplistElement;
+        snaplist.waitForVisible();
+        assert.isNotNull(snaplist);
```

Comment:
isNotNull assertion is not needed, this will never fail anyway as snaplist is a jason object. same for other tests below